CONTRIBUTING.md

```diff
@@ -156,7 +217,8 @@ When writing new or refactoring old code please make the following changes.
     }
     ```
 
-  * Fix [`std::string::c_str()`](http://en.cppreference.com/w/cpp/string/basic_string/c_str) calls where passed to [`std::string`](http://en.cppreference.com/w/cpp/string/basic_string) parameters.
+  * Fix [`std::string::c_str()`](http://en.cppreference.com/w/cpp/string/basic_string/c_str) calls where passed to 
```

Comment:
When I view this section using macdown, this bullet and those that follow are indented under the preceding text. I'm not sure if this is what you intended. LMK if you'd like me to reformat this section.

![Screen Shot 2021-03-02 at 9 53 18 AM](https://user-images.githubusercontent.com/13138489/109692687-abc57c80-7b3d-11eb-8ef4-58978b7fa009.png)

Reply:
If you use the "Rich Diff" button in the the GitHub file view, the indentation doesn't occur.  I think you're seeing a difference with between GitHub and macdown markdown

Reply:
I see what you mean -- macdown wants a lot more whitespace.
OK, close enough for rock'n'roll. I'll drop this concern.

Reply:
For some reason there was a single space before `*` on a few lines then two spaces on the others. I have corrected so we will see if they display consistently now.